APE1.rst

```diff
@@ -1,11 +1,11 @@
 APE Purpose and Process
 -----------------------
 
-author: Perry Greenfield
+author: Perry Greenfield, Lia Corrales
```

Comment:
Minor: should we keep the author of modifications separate, e.g by saying e.g. **Addendum: July 22, 2020 (author: Lia Corrales)** below?